Protocol.md

```diff
@@ -56,15 +56,26 @@ If both parties already have this information, transmission is unnecessary.
 #### A's fund amount (satoshi)
 How much A inputs into the contract.
 
-#### A's extended public key
-The extended public key to be used to [derive](https://github.com/discreetlogcontracts/dlcspecs/blob/master/KeyDerivation.md) public keys for the Fund transaction, refund transaction and CETs.
+#### A's public keys
+The following public keys:
+
+1. Funding Public Key
```

Comment:
I feel it wouldn't hurt to add a small descriptions to theses as you've done for the addresses (especially for the second one I feel it would help).